components/root_layout/root_layout_body.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <div
     ref="root-layout-body"
-    :class="['root-layout__body', 'd-fl-grow1', bodyClasses]"
+    :class="['root-layout__body', { 'd-of-y-auto': isInMobileMode }, 'd-fl-grow1', bodyClasses]"
```

Comment:
One suggestion to try to clean up a bit the template is to apply a `stacked` modifier class to the body and define the overflow styles based on this class. Example:

```css
.root-layout__body {
  &.root-layout__body--stacked {
    overflow-y: auto;
    
    .root-layout__sidebar, .root-layout__content {
      overflow-y: unset
    }
  }
}

.root-layout__sidebar, .root-layout__content {
  overflow-y: auto;
}
```

Reply:
I guess the `height` can also be added in these CSS classes if they're defined in the component with https://vuejs.org/api/sfc-css-features.html#v-bind-in-css